src/mlpack/core/tree/binary_space_tree/traits.hpp

```diff
@@ -73,7 +73,6 @@ class TreeTraits<BinarySpaceTree<MetricType, StatisticType, MatType,
 {
  public:
   static const bool HasOverlappingChildren = true;
-  static const bool HasDuplicatedPoints = false;
```

Comment:
I think it would be useful to keep this trait, even if it is not used right now.